chore: add risks of passkeys for sso (#320)
matter-labs/zksync-docs: content/00.zksync-era/30.unique-features/30.zksync-sso/24.passkeys.md

which support various authenticators like built-in device biometrics (fingerprint sensors, facial recognition)
or external hardware security keys (YubiKey, SoloKeys). This ensures a high level of security and a seamless user experience.

## What are the risks and challenges of Passkeys?

Passkeys come with a set of challenges and risks with passkey adoption, including:

> - **Inconsistent support and experiences**: Due to the history of their development, there are currently multiple ‘flavours’ of passkey available that providers and users need to understand how to manage. These range from device-bound and physical token passkeys (that never leave the device) to ‘synced’ passkeys (where a device’s Credential Manager backs up and synchronises passkeys across the user’s other devices).

Author: typo: 
```suggestion
- **Inconsistent support and experiences**: Due to the history of their development, there are currently multiple ‘flavours’ of passkey available that providers and users need to understand how to manage. These range from device-bound and physical token passkeys (that never leave the device) to ‘synced’ passkeys (where a device’s Credential Manager backs up and synchronize passkeys across the user’s other devices).
```

> - **Device loss scenarios**: Backing up and synchronising passkeys with a Credential Manager makes it easier to recover access to them compared to other existing second factor options. However, this relies on the user having prepared their Credential Manager account for recovery. Users need help in understanding and implementing the right steps so they can feel ready to go passwordless and use passkeys without extra worry and hassle.

Author: typo:
```suggestion
- **Device loss scenarios**: Using a Credential Manager to synchronize and back up passkeys makes it easier to recover access to them compared to other existing second factor options. However, this relies on the user having prepared their Credential Manager account for recovery. Users need help in understanding and implementing the right steps so they can feel ready to go passwordless and use passkeys without extra worry and hassle.
```

which support various authenticators like built-in device biometrics (fingerprint sensors, facial recognition)
or external hardware security keys (YubiKey, SoloKeys). This ensures a high level of security and a seamless user experience.

## What are the risks and challenges of Passkeys?

Passkeys come with a set of challenges and risks with passkey adoption, including:

- **Inconsistent support and experiences**: Due to the history of their development, there are currently multiple ‘flavours’ of passkey available that
providers and users need to understand how to manage. These range from device-bound and physical token passkeys (that never leave the device) to

> ‘synced’ passkeys (where a device’s Credential Manager backs up and synchronize passkeys across the user’s other devices).

Author: additional info:
```suggestion
‘synced’ passkeys (where a device’s Credential Manager backs up and synchronize passkeys across the user’s other devices). For example, Apple
passkeys are automatically backed up to the cloud to be synced across all your Apple devices, so compromising this account can lead to token access.
```

which support various authenticators like built-in device biometrics (fingerprint sensors, facial recognition)
or external hardware security keys (YubiKey, SoloKeys). This ensures a high level of security and a seamless user experience.

## What are the risks and challenges of Passkeys?

Passkeys come with a set of challenges and risks with passkey adoption, including:

- **Inconsistent support and experiences**: Due to the history of their development, there are currently multiple ‘flavours’ of passkey available that
providers and users need to understand how to manage. These range from device-bound and physical token passkeys (that never leave the device) to
‘synced’ passkeys (where a device’s Credential Manager backs up and synchronize passkeys across the user’s other devices). For example, Apple
passkeys are automatically backed up to the cloud to be synced across all your Apple devices, so compromising this account can lead to token access.

> passkeys are automatically backed up to the cloud to be synced across all your Apple devices, so compromising this account can lead to token access.

Author: typo: line is duplicated
```suggestion
```

which support various authenticators like built-in device biometrics (fingerprint sensors, facial recognition)
or external hardware security keys (YubiKey, SoloKeys). This ensures a high level of security and a seamless user experience.

## What are the risks and challenges of Passkeys?

Passkeys enable the simplification of the UI/UX to sign blockchain transactions securely and across multiple platforms. While it improves usability,

> Passkeys security relies on platform integrity, account protection, and domain ownership. Users should remain aware of these risks when implementing

Reviewer: ```suggestion
Passkeys security relies on platform integrity, account protection, and domain ownership. Users should remain aware of these risks when adopting
```

users don't usually implement

passkey-based authentication.

- **Passkeys Are Still Cryptographic Keys**: Unauthorized access to passkey material allows attackers to sign transactions, similar to a compromised
private key.
- **Passkeys Security Is Vendor-Specific**: Apple, Google, Microsoft, and other platforms implement passkeys differently, with varying security models
tied to device and account protections, which users need to understand. These range from device-bound and physical token passkeys (that never leave
the device) to 'synced' passkeys (where a device's Credential Manager backs up and synchronize passkeys across the user's other devices). For example,

> Apple passkeys are automatically backed up to the cloud to be synced across all your Apple devices, so compromising Apple Account can lead to token access.

Reviewer: ```suggestion
Apple passkeys are automatically backed up to the cloud to be synced across all your Apple devices, so compromising your Apple Account can lead to token access.
```

- **Passkeys Are Tied to Identity and Devices**: The security of passkeys depends on the protection of associated accounts (e.g., Apple ID, Google

> Account) and the devices where they are stored. Users might need help in understanding and implementing the right security practices (MFA, Strong

Reviewer: ```suggestion
Account) and the devices where they are stored. Users might need help in understanding and implementing the right security practices (MFA, strong
```

pin-codes and passwords) so they can feel ready to go passwordless and use passkeys without extra worry and hassle.

> - **Domain Ownership Risk**: Passkeys are linked to specific domains usually maintained by an application developers. If a domain is lost, hijacked,

Reviewer: ```suggestion
- **Domain Ownership Risk**: Passkeys are linked to specific domains usually maintained by an application developer. If a domain is lost, hijacked,
```

> or transferred, authentication could be compromised, meaning that users could be at risk of authorizing transactions with unknown consequences,

Reviewer: ```suggestion
or transferred, authentication could be compromised.
This means that users could be at risk of authorizing transactions with unknown consequences,
```
